crates/api_common/src/post.rs

```diff
@@ -78,6 +78,9 @@ pub struct GetPosts {
   pub liked_only: Option<bool>,
   pub disliked_only: Option<bool>,
   pub moderator_view: Option<bool>,
+  // optional when_* parameters are in milliseconds since 00:00:00 UTC on 1 January 1970
```

Comment:
Use `///` for each of these two fields. And the comments can just specify that its unix time millis.

Reply:
> And the comments can just specify that its unix time millis.

The sparsity of comments I feel like has been a barrier to new developers understanding the Rust and Diesel code. I've a professional technical writer, authored academic teaching books, and have been a in-person teacher for many major companies in the computer industry. I really think verbose is better than terse...